projects/geoengine-app/src/index.html

```diff
@@ -6,6 +6,9 @@
 
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" type="image/x-icon" href="favicon.ico" />
+
```

Comment:
is this required? can't npm handle the imports?

Reply:
I found a better solution
5facb734d8279dcd5b9f8cdd9091473fcb94f486